annotations/src/lib.rs

```diff
@@ -1065,17 +1065,17 @@ pub fn mirai_abstract_value<T>(x: T) -> T {
 
 // Helper function for MIRAI. Should only be called via the add_tag! macro.
 #[doc(hidden)]
-pub fn mirai_add_tag<V, T>(_v: &V) {}
+pub fn mirai_add_tag<V: ?Sized, T>(_v: &V) {}
```

Comment:
There are more instances of this in the annotations crate. You may as well fix them too.

Reply:
I had a look at other instances but they are different because an unsized type needs to be used behind some kind of pointer.